cmd/gateway/azure/gateway-azure-anonymous.go

```diff
@@ -28,6 +29,7 @@ import (
 	"time"
 
 	"github.com/Azure/azure-sdk-for-go/storage"
+	minio "github.com/minio/minio/cmd"
```

Comment:
`minio` alias confuses with `minio-go`.  you could keep it as `cmd` itself

Reply:
The reason i kept was using `cmd.` is just not cleanly readable.